rules/falco_rules.yaml

```
@@ -2414,7 +2421,9 @@
     (container.image.repository in (gcr.io/google_containers/hyperkube-amd64,
      gcr.io/google_containers/kube2sky, docker.io/sysdig/falco,
      docker.io/sysdig/sysdig, docker.io/falcosecurity/falco,
-     sysdig/falco, sysdig/sysdig, falcosecurity/falco) or (k8s.ns.name = "kube-system"))
+     sysdig/falco, sysdig/sysdig, falcosecurity/falco)
+     or (container.image.repository in ibm_cloud_containers)
```

```suggestion
     or (container.image.repository in (ibm_cloud_containers))
```
IIRC, this :point_up:  should solve the issue.

---

Ah thanks @leogr

---

Hrm I thought I'd fixed this. I'll check tomorrow.